From ca6bee75c9d5228711d429cc632baa93171faf57 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Wed, 23 Sep 2020 11:18:13 +0800
Subject: [PATCH] Test case for Delete SecondTestFile.h in FrontPage.h file.

Description: Delete SecondTestFile.h in the MdeModulePkg/Application/UiApp/FrontPage.h

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdeModulePkg/Application/UiApp/FrontPage.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/MdeModulePkg/Application/UiApp/FrontPage.h b/MdeModulePkg/Application/UiApp/FrontPage.h
index a29a17987e..08c0649bb7 100644
--- a/MdeModulePkg/Application/UiApp/FrontPage.h
+++ b/MdeModulePkg/Application/UiApp/FrontPage.h
@@ -11,7 +11,6 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #include "String.h"
 #include "Ui.h"
-#include "SecondTestFile.h"
 
 #include <Protocol/BootLogo.h>
 //
@@ -40,7 +39,7 @@ extern EFI_FORM_BROWSER2_PROTOCOL      *gFormBrowser2;
 
 #define FRONT_PAGE_FORMSET_GUID \
   { \
-    front_page_guid, 0x3f06, 0x4ba6, {0x82, 0x88, 0x9, 0x17, 0x9b, 0x85, 0x5d, 0xbe} \
+    0x9d0b30db, 0x3f06, 0x4ba6, {0x82, 0x88, 0x9, 0x17, 0x9b, 0x85, 0x5d, 0xbe} \
   }
 
 #define FRONT_PAGE_CALLBACK_DATA_SIGNATURE  SIGNATURE_32 ('F', 'P', 'C', 'B')
-- 
2.27.0.windows.1

